src/utmpx.rs

```diff
@@ -238,6 +270,7 @@ mod tests {
         unsafe { libc::endutxent() };
     }
 
+    #[cfg(any(target_env = "gnu", target_os = "dragonfly", target_os = "freebsd"))]
     #[test]
     /// Test a bare get entry.
     fn test_getutxent() {
```

Comment:
What coverage does this test provide that the integration test doesn't?

Reply:
No additional coverage. But this helps sanity checking the C primitives, especially on the CI here where I don't have insights otherwise.

Reply:
I think this test should be deleted, then.  It's not Nix's job to write a test suite for the utmpx(3) functions.  The integration test is good enough for Nix's purposes.

Reply:
Ack. Should I delete all the unit tests then? Or were you specifically concerned with this one only?

Reply:
Yeah, you should probably delete them all.